commands/agents.md

```diff
@@ -0,0 +1,125 @@
+
+---
+
+# TASK
+
+Your task is to analyze the architecture file at `context/product/architecture.md` and create domain expert subagent files that match the project's technical needs. You will compare the architecture against existing subagents to identify gaps, propose new subagents with clear justification, and create complete agent files in `.claude/agents/` for each approved agent.
+
+---
+
+# INPUTS & OUTPUTS
+
+- **Prerequisite Input:** `context/product/architecture.md` (The technical blueprint - must exist).
+- **Context Input:** `context/product/product-definition.md` (For understanding the domain).
+- **Reference Input:** `.claude/agents/` (Existing subagent examples for pattern reference).
+- **Template File:** `.awos/templates/agent-template.md` (Subagent file format and best practices).
+- **Output Files:** `.claude/agents/[agent-name].md` (Complete agent file with config and system prompt)
+
+---
+
+# PROCESS
+
+Follow this logic precisely!
+When you need user input on a decision:
+
+- Use **AskUserQuestion** tool with clear, clickable options
+- Never present numbered lists requiring manual number entry
+
+### Step 1: Prerequisite Checks
+
+- First, check if `context/product/architecture.md` exists.
+- If the file is missing, stop immediately. Respond with: "Before we can suggest domain expert subagents, we need a defined architecture. Please run `/awos:architecture` first, then run me again."
+- If the file exists, proceed to the next step.
+
+### Step 2: Architecture Analysis
+
+1. **Read and Analyze:**
+   - Announce the task: "I will now analyze your architecture to identify the technologies and patterns that would benefit from specialized domain expert subagents."
+   - Read `context/product/architecture.md` carefully.
+   - Optionally read `context/product/product-definition.md` for additional domain context.
+
+2. **Extract Key Technologies:**
+   - Identify all **programming languages** (e.g., TypeScript, Python, Kotlin, Go, Rust).
+   - Identify all **frameworks** (e.g., React, FastAPI, Spring Boot, Next.js).
+   - Identify all **databases** (e.g., PostgreSQL, MongoDB, Redis, Elasticsearch).
+   - Identify all **infrastructure components** (e.g., Kubernetes, AWS, Docker, Terraform).
+   - Identify all **architectural patterns** (e.g., microservices, event-driven, serverless).
+
+3. **Present Analysis:**
+   - Summarize what you found in a clear format:
+     ```
+     Tech Stack Analysis:
+     - Languages: [list]
+     - Frameworks: [list]
+     - Databases: [list]
+     - Infrastructure: [list]
+     - Patterns: [list]
+     ```
+
+### Step 3: Existing Subagent Check
+
+1. **Scan Existing Subagents:**
+   - Check what subagent files already exist in `.claude/agents/` (including any subdirectories).
```

Comment:
Same question. What if you use agents from a plugin? We cannot use non-awos specific routes